packages/@atjson/offset-annotations/src/index.ts

```diff
@@ -88,5 +90,7 @@ export default class OffsetSource extends Document {
     TwitterEmbed,
     Underline,
     VideoEmbed,
+    Accordion,
+    AccordionGroup,
```

Comment:
Sorry to be a tad annoying, but could you alphabetize the blocks in the schema? 😅

Thanks!!

Reply:
alphabetized pls review